services/web-app/pages/about-carbon/how-carbon-works.mdx

```diff
@@ -67,25 +67,26 @@ as color tokens, type tokens, spacing tokens, the grid, and more.
 **Guidelines** provide direction on critical topics such as accessibility and content and ensure
 consistent user experiences that meet IBM's standards.
 
-**Assets** include re–usable resources such as components, patterns, functions, and templates. Each
-asset has code, designs, and usage guidance.
+**Assets** are re–usable resources such as components, patterns, functions, and templates. Assets
+often have usage and accessibililty guidance, as well as styling specifications and code.
 
-**Design kits** are libraries from Sketch and Figma—and other tools such as Adobe XD and Invision.
-They contain components and elements as well as wireframes and guidance to aid designers.
+**Design kits** contain elements, guidelines, user interface components and patterns, and wireframes
```

Comment:
This feels like its describing what kinds of kits we have instead of what a design kit is. The last part is getting at its essence more. 

Here is my stab at it feel free to take what you like from it.

"Design kits are the connected or re-usable design files associated with certain coded assets or libraries used to speed up the design creation process. They are available in various design tools such as Sketch and Figma."

Reply:
Follow up question: are design kits assets? Or do design kits contain assets?

Reply:
Per our schema, no, design kits and assets are different types of resources (have different structured data and different relationships with other resources).

Maybe this is me, but I view an "asset" as a resource that get's installed and directly used in published and released digital experiences.

"Design kits" are more like a tool... a means to create design specifications (e.g. redlines) which are more of a deliverable, and design assets (e.g. images that get used in digital experiences).

Reply:
I reverted this so the description reads as it did before, using "library" to describe "design kit".